assets/styles.css

```diff
@@ -0,0 +1,712 @@
+  min-height: 280px;
+}
+
+.category .content::before {
+  content: '';
+  height: 100%;
+  background: rgb(255, 255, 255);
+  background: linear-gradient(180deg, rgba(255, 255, 255, 0.1) 0%, rgba(0, 0, 0, 1) 100%);
+}
+
+.content-section {
+  bottom: 0;
+  left: 0;
+  padding: 14px;
+  color: #fff;
+  z-index: 2;
+}
+
+.content-section h2 {
+  font-size: 20px;
+}
+
+.content-section small {
+  top: -10px;
+  color: #f5f5f5;
+}
+
+.category .content2 {
+  background-color: #fff5f4;
+  align-items: center;
+}
+
+.content2 article {
+  padding: 15px 0 15px 15px;
+  color: red;
+}
+
+.categories h5 {
+  text-transform: uppercase;
+  font-weight: 300;
+}
+
+.content2 .article-banner {
+  flex-basis: 96px;
+  height: 96px;
+  flex-shrink: 0;
+  background-color: red;
+  background-image: url('https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/08/astronautpoop.jpg');
+  background-position: center;
+}
+
+.categories .others {
+  margin-left: -40px;
+  font-size: 13px;
+}
+
+.others li {
+  line-height: 1.6em;
+  padding: 12px 0;
+  border-bottom: solid thin lightgray;
+}
+
+.others li::before {
+  content: "\2022";
+  color: red;
+  font-weight: bold;
+  display: inline-block;
+  width: 1em;
+}
+
+.latest-funding-cards {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(270px, 1fr));
+  gap: 20px;
+}
+
+.latest-funding-card {
+  height: 250px;
+}
+
+.card-img img {
+  max-width: 100%;
+  width: 60px;
+  height: 60px;
+  top: 7px;
+  left: 7px;
+}
+
+.card-header h4 {
+  font-size: 21px;
+  font-weight: 900;
+  text-align: center;
+  line-height: 20px;
+}
+
+.card-title {
+  background-color: #eb4222;
+  color: #fff;
+  padding: 1rem;
+}
+
+.card-data {
+  padding: 0.31rem 0;
+  text-align: center;
+  background-color: #fcebe6;
+  line-height: 20px;
+}
+
+.card-data h5 {
+  font-weight: 200;
+}
+
+.more-card-details {
+  border-top: 1px solid blanchedalmond;
+}
+
+.card-details,
+.more-card-details {
+  font-size: 15px;
+  padding: 0.5rem;
+}
+
+.all-deals-link {
+  float: right;
+  top: -20px;
+  font-size: 0.9em;
+  color: red;
+}
+
+.slash {
+  text-decoration: line-through;
+}
+
+.deal-price {
+  font-size: 1.2em;
+  color: #333;
+}
+
+.sub-footer,
+.main-footer {
+  min-height: 30px;
+}
+
+.sub-footer {
+  background-color: #131313;
+  color: #fff;
+  text-align: center;
+  padding: 30px 0;
+}
+
+.sub-footer ul {
+  padding: 10px 0;
+  margin: auto;
+}
+
+.sub-footer ul li {
+  display: inline-block;
+  padding: 5px;
+}
+
+.sub-footer ul:first-child li {
+  width: 40px;
+  height: 40px;
+  border-radius: 20px;
+  margin: 0 5px;
+  background-color: #333;
+  border: solid thin #333;
+  color: #fff;
+}
+
+.sub-footer ul:first-child li a {
+  top: 2px;
+}
+
+.sub-footer ul:last-child li {
+  text-transform: uppercase;
+  font-size: 11px;
+}
+
+.sub-footer ul:first-child li:hover {
+  background: transparent;
+}
+
+.main-footer {
+  background-color: #000;
+  color: #fff;
+  font-size: 13px;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `color: #fff;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.white-color{
  color: #fff;
}
```
**Html file**

```
<div class="white-color">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_